content/blog/cosign-2.5.0.md

```diff
@@ -0,0 +1,132 @@
+---
+title: "Introducing the Harbor and Cosgin in v2.5.0"
+author:
+  name: "Orlin Vasilev"
+description: "Harbor and cosign integration"
+date: 2022-04-07T12:00:00+04:00
+---
+![Cosign](https://raw.githubusercontent.com/sigstore/cosign/main/images/logo.svg)
+
+TL;DR;  
+Artifact signing and signature verification are critical security capabilities that allow you to verify the integrity of an artifact. Harbor supports content trust through integrations with [Notary](https://github.com/notaryproject/notary) and [Cosign](https://github.com/sigstore/cosign).  
+
+Harbor v2.5 integrates support for [Cosign](https://github.com/sigstore/cosign), a OCI artifact signing and verification solution that is part of the [Sigstore project](https://github.com/sigstore).
+
+Cosign signs OCI artifacts and pushes the generated signature into Harbor. This signature is stored as an artifact accessory along side the signed artifact. Harbor manages a link between the signed artifact and cosign signature, allowing you to apply things like **tag retention rules** and **immutable rules** to a signed artifact, and it will extend to both the signed artifact and the signature. In this way you can use Harbor's built in functionality to manage signed artifacts and Cosign signature accessories.
+
+A key feature of using Cosign with Harbor is the ability use Harbor's [replication capabilities](https://goharbor.io/docs/2.5.0/administration/configuring-replication/) to replicate signatures with their associated signed artifact. This means that if a **replication rule** applies to a signed artifact, Harbor will apply the replication rule to the signature in the same way it applies it to the signed artifact.
+
+* When replicating between Harbor instances, the target Harbor instance will maintain the link between the signed artifact and its associated signatures. You will be able to see the relationship between the two artifacts in the target Harbor interface, in the same way that you do in the source registry.
+
+Full Documentation you will be able to find [here](https://goharbor.io/docs/2.5.0/working-with-projects/working-with-images/sign-images/)
+
+## Demo Setup 
+**Configure two Harbor instances with Cosign enabled per repo and configure replication**
+
+#### 1. Install two instances with cosig(and notary) enabled
+   In our setup we use [offline installer](https://goharbor.io/docs/2.5.0/install-config/download-installer/).
```

Comment:
it shows 404 on https://goharbor.io/docs/2.5.0/install-config/download-installer/

Reply:
that because we haven't pushed the new site and documentation online when we release Monday 11th will be working :)